frontend/src/components/common/Form/Input/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { FormGroup, InputGroup, Intent } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { debounce } from "lodash-es";
+import React, { FC, useRef, useState } from "react";
+import { DEBOUNCE_TIME_MS } from "../../../CreateCollectionModal/components/Content/common/constants";
+import { Value } from "../common/constants";
+import { LabelText, StyledIcon, StyledLabel } from "./style";
+
+interface Props {
+  name: string;
+  text: string;
+  percentage?: number;
+  disabled?: boolean;
+  handleChange: ({ isValid, value, name }: Value) => void;
+  syncValidation?: Array<(value: string) => true | string>;
+  noNameAttr?: boolean;
+  className?: string;
+  placeholder?: string;
+}
+
+const DangerIcon = () => {
+  return <StyledIcon icon={IconNames.ISSUE} intent={Intent.DANGER} />;
+};
```

Comment:
should this be a directory level constant?

Reply:
Ah I see that it's already defined at the top level of this file, so we're not creating a new function per re-render. What do you mean by directory level constant 😄 ? Thank you!

Reply:
I saw the same snippet in a few places,might be worth pulling out into its own file

Reply:
Oh I see! Yeah there are two places that use this, but their `StyledIcon` actually has different styles!